src/basic/mount-util.c

```diff
@@ -290,6 +291,7 @@ int path_is_mount_point(const char *t, const char *root, int flags) {
                 if (r < 0)
                         return r;
 
+                log_debug("canical: %s → %s", t, canonical);
```

Comment:
"canical"?

Reply:
Ooops, this wasn't supposed to land in the commit ;(